crates/bevy_app/src/app.rs

```diff
@@ -568,7 +568,11 @@ impl App {
         self
     }
 
-    pub fn add_sub_app(&mut self, app: App, f: impl Fn(&mut World, &mut App) + 'static) -> &mut Self {
+    pub fn add_sub_app(
```

Comment:
`cargo fmt` changes to some files..

Reply:
proper formatting in a prototype? get that out of here :smile: